ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/entities/FragmentInfo.java

```diff
@@ -3,28 +3,20 @@
 public class FragmentInfo {
     private final String view;
     private final String shape;
-    private final String parentFragmentId;
     private final String viewShortName;
     private final String path;
     private String value;
-    private Long memberLimit;
```

Comment:
What is the reasoning for removing the memberlimit and the parent? 
The parent is part of the design for the future, but I can understand why you prefer to only introduce that whilst we need it.
I presume you removed the limit since you only use the viewconfig?

Reply:
Yep, those were my concerns :-)